Frontend/src/app/feed/feed.component.ts

```diff
@@ -12,7 +14,8 @@ import { DomSanitizer } from '@angular/platform-browser';
 export class FeedComponent {
```

Comment:
it reflecting error at this point, check this onece

Reply:
This is related to the issue mentioned above